modules/lifestreetBidAdapter.js

```diff
@@ -0,0 +1,143 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please pull in the specific functions needed here

Reply:
done